content.js

```diff
@@ -1,10 +1,43 @@
 setTimeout(function onLoad() {
     if (typeof $ != 'undefined') {
-        if ($(".js-link:contains('View all activities')").length > 0) {
-            $(".js-link:contains('View all activities')")[0].click();
-        }
-        $(".js-link:contains('View activities with bookings')").hide();
-    }
-
-    setTimeout(onLoad, 1000);
-}, 1000);
+
+      console.log("Peek Fixer Monitor")
+
+      // ----======= Updating Dashboard Page =======----
+
+      // Clicking 'View all activities' link
+      if ($(".js-link:contains('View all activities')").length > 0) {
```

Comment:
For those selectors you use more than once, it would be a lot more performant if you put it in a var and use that.

`var viewAllActivitiesLink = $(".js-link:contains('View all activities')")`

then 

```
if (viewAllActivitiesLink.length > 0) {
    viewAllActivitiesLink[0].click();
    viewAllActivitiesLink.hide();
}
```